app/Http/Livewire/View/CalendarlistComponent.php

```diff
@@ -45,11 +45,13 @@ class CalendarlistComponent extends Component
     public $closeModal = false;
     public $callbackModal = false;
 
+    public $format = 'd.m.Y';
```

Comment:
different from Y-m-d

Reply:
This is assigning a standard value before receiving JS data from the browser